crates/wdk-build/rust-driver-makefile.toml

```diff
@@ -283,11 +395,12 @@ wdk_build::cargo_make::copy_to_driver_package_folder(
 
 [tasks.inf2cat]
 private = true
-dependencies = ["copy-sys-to-package", "copy-inf-to-package"]
+dependencies = ["copy-driver-binary-to-package", "copy-inf-to-package"]
+env = { "WDK_BUILD_INF2CAT_OS" = { source = "${CARGO_MAKE_CRATE_TARGET_TRIPLE}", default_value = "UNKNOWN", mapping = { "x86_64-pc-windows-msvc" = "10_x64", "aarch64-pc-windows-msvc" = "Server10_arm64" } } }
```

Comment:
I used `10_x64` as that was the default value that the WDK resolves to when building C drivers. 

Is there a new value every WDK release (I see `NI` in your suggestion)? If so, there'd need to be some logic here to resolve the mapping here based on the WDK config.

I'm inclined to leave it as is for this PR, since it matches parity with building C drivers

Reply:
When you say the default value, did you get that from WDK props, or...?

In any event see https://learn.microsoft.com/en-us/windows-hardware/drivers/devtest/inf2cat here for the full list of supported OS flags.  You can also provide multiple values to validate for multiple different OSes.  I think at the very least we should go to 10_NI for our default here since Rust drivers are at this early stage experimental and based on our latest OS planning.

Reply:
This particular inf2cat setting corresponds with the `Inf2CatWindowsVersionList` property. 

On arm64, it defaults to `Server10_$(DDKPlatform)` where `DDKPlatform` resolves to `arm64`.
On x64, it defaults to `10_$(DDKPlatform)` where `DDKPlatform` resolves to `x64`.
In GE WDK, these defaults are set in `content\Program Files\Windows Kits\10\build\10.0.26100.0\WindowsDriver.OS.Props`.

If you open one of the sample windows drivers (ex. `umdf2toaster.sln`) in VS22, it confirms the above:
**x64:**
![image](https://github.com/user-attachments/assets/77c15645-247f-417d-b16a-bc9634282540)
![image](https://github.com/user-attachments/assets/14308a2d-3715-4201-b911-ce3a98787f66)
**arm64**
![image](https://github.com/user-attachments/assets/ed8fdcb2-29ab-4755-ac2f-cd8a180dd0dc)
![image](https://github.com/user-attachments/assets/e70060f3-5519-4d94-a4dd-30d85be87ad0)

Reply:
For others reading this review: we resolved this offline and this is not blocking.